qiita_pet/templates/study_ajax/study_analyses.html

```diff
@@ -0,0 +1,44 @@
+{% from qiita_core.qiita_settings import qiita_config %}
+
+<script type="text/javascript" src="{% raw qiita_config.portal_dir %}/static/vendor/js/jquery.validate.min.js"></script>
+<script type="text/javascript>">
+  $('#studies-analyses-table').dataTable();
+</script>
+<h3>Analyses that have used this study</h3>
+
+Note that if an analysis doesn't have a link, it means that is a "User default analysis"; which means that the analysis belows to that user but the user has only selected artifacts but hasn't created the actual anlaysis. Please contact the user and ask them to delete that artifact from their analysis creation page.
```

Comment:
This seems like a pretty extreme edge-case. Should the analysis creation just error-out for a user that had an artifact in their cart but "lost" it because it was deleted?

```suggestion
Note that if an analysis doesn't have a link, it means that is a "User default analysis"; which means that the analysis belongs to that user but the user has only selected artifacts and hasn't created the actual analysis. Please contact the user and ask them to delete that artifact from their analysis creation page.
```